### DISCUSSION TITLE
Trino version 402 no data preview in superset using UI

### DISCUSSION THREAD

**Discussion by rnpaiva** [trinodb/trino#14992] [Q&A]
Steps:
- Install superset 2.0.0
- Install trino version 400 with redshift connector 
- Configure it and connect to redshift

Expected results :
- Select in superset the connection [ok]
- Schema is loaded in UI [ok]
- Wrote the query in query editor and run [ok] 
- Table is loaded in UI [not loading]

Conclusion : 
- If I downgrade trino to version 391 I'm able to see the tables in superset UI .

**Comment by ebyhr** [comment]
Can you find the first version which caused the issue?
